app/Http/Middleware/Role.php

```diff
@@ -16,22 +16,19 @@ class Role
      */
     public function handle(Request $request, Closure $next): Response
     {
-        if(!Auth::check()){
+        if (!Auth::check()) {
             return redirect()->route('loginScreen');
-        }else{    
-            $user=Auth::user();
-            if($user->role == 1){
+        } else {
+            $user = Auth::user();
+            if ($user->role == 1) {
```

Comment:
Advice : Instead of using same code repeatedly across the project,
you can add new function in user.php in order to check current auth user is admin. 

```
public function isAdmin()
{
    return $this->role == config('constants.user_role.admin_no');
}
```

And then you can access that via Auth::user()->isAdmin() from anywhere.